code/datums/ai/_ai_controller.dm

```diff
@@ -109,21 +109,26 @@ have ways of interacting with a specific atom and control it. They posses a blac
 		if(behavior_cooldowns[current_behavior] > world.time) //Still on cooldown
 			continue
 
+		// Convert the current behaviour action cooldown to realtime seconds from deciseconds.current_behavior
+		// Then pick the max of this and the delta_time passed to ai_controller.process()
+		// Action cooldowns cannot happen faster than delta_time, so delta_time should be the value used in this scenario.
```

Comment:
in the future I will probably separate the planning and behavior subsystems so that this isn't as much of an issue.